kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/JRF/mii-initial-d1-JRF-v1.yaml

```diff
@@ -93,7 +93,7 @@ spec:
       - name: USER_MEM_ARGS
         value: "-Djava.security.egd=file:/dev/./urandom -Xms512m -Xmx1024m "
    
-  # The number of managed servers to start for unlisted clusters
+  # The number of admin servers to start for unlisted clusters
```

Comment:
This edit is incorrect.  replicas controls managed servers, not admin servers.

Reply:
@tbarnes-us I do not understand why `admin` is not the correct server type here, because this comment pertains to the `adminServer` on line 80. Or am I mistaken?

Reply:
The comment applies to the 'replicas:' attribute below, which is the default replica count for all clusters (`kubectl explain domain.spec.replicas`).